web_responsive/static/src/css/web_responsive.scss

```
@@ -421,20 +425,21 @@ html .o_web_client .o_main .o_main_content {
                     min-width: 30%;
                     overflow: auto;
 
-                    .o_chatter_topbar,
-                    .o_thread_composer {
-                        position: sticky;
-                    }
-
-                    .o_chatter_topbar {
+                    .o_chatter_header_container {
+                        padding-top: $grid-gutter-width*0.5;
```

Add beautiful spacing please :relaxed: 

```suggestion
                        padding-top: $grid-gutter-width * 0.5;
```